lib/config.js

```diff
@@ -228,9 +228,11 @@ const getFullConfig = async ({
 
   if (pkgConfig.ciVersions) {
     let versions = pkgConfig.ciVersions
-    if (versions === 'latest') {
-      const defaultVersions = [rootPkgConfig, defaultConfig].find(c => Array.isArray(c.ciVersions))
-      versions = defaultVersions.ciVersions.slice(-1)
+    if (versions.includes('latest')) {
```

Comment:
The only way this would have worked before is if we supported this being a string `latest`, which we don't anymore?

Reply:
We still support that, and it's used by the docs workspace in the CLI, so I didn't want to break that.

Reply:
Updated the code to make it correctly only match the string "latest" or an array containing that string.